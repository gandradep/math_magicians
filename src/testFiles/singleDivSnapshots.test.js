import renderer from 'react-test-renderer';
import CalcButtons from '../components/SingleDiv';

it('SingleDiv renders correctly', () => {
  const tree = renderer
    .create(<CalcButtons />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
