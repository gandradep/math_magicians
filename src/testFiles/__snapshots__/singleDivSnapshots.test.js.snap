// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleDiv renders correctly 1`] = `
<button
  onClick={[Function]}
  type="button"
/>
`;
